lib/interface.rb

```diff
@@ -0,0 +1,96 @@
+class Interface
+
+  attr_reader :menu_array, :order
+
+  def initialize(filepath)
+    @menu = Menu.new(filepath)
+    @order = Order.new
+    @order_array = []
```

Comment:
It's idiomatic in Ruby to not describe the type of a variable unless it's absolutely necessary for clarity. `@orders` would be clear enough.